ui/src/app/flags/Flag.tsx

```diff
@@ -157,12 +161,38 @@ export default function Flag() {
               )}
               {flagTypeToLabel(flag.type)}
             </div>
+            {info.analytics?.enabled && (
+              <button
+                className="ml-2 p-1 rounded hover:bg-accent"
+                title="View Analytics"
+                onClick={() =>
+                  navigate(
+                    `/namespaces/${namespace.key}/analytics?flag=${flag.key}`
+                  )
```

Comment:
is there any blocker to make url like `/namespaces/$(ns)/analytics/${flag.key}`?

Reply:
Yeah I will update it to be like that. That way it works like /flags /segments etc

Reply:
done